components/dashboard/src/start/VerifyModal.tsx

```diff
@@ -93,8 +95,8 @@ export function VerifyModal() {
                 ) : (
                     <></>
                 )}
-                <div className="mt-4">
-                    <h4>Mobile Phone Number</h4>
+
+                <InputField label="Mobile Phone Number">
```

Comment:
This one is tricky to test w/o hacking in a render for it directly. I did this and verified it still works as expected.